tap_stripe/__init__.py

```diff
@@ -85,6 +85,9 @@
     'transfers': {'type': 'transfer.*', 'object': ['transfer']},
     'disputes': {'type': 'charge.dispute.*', 'object': ['dispute']},
     'products': {'type': 'product.*', 'object': ['product']},
+    'invoice_line_items': {'type': 'invoice.*', 'object': ['invoice']},
```

Comment:
Can we use object as "line_item" for sub-stream "invoice_line_items" stream? Same for below sub-stream "subscription_items".

Reply:
Done